### PR TITLE
[Rel v0.2] Migrate leftovers

### PR DESCRIPTION
**Pull request by gkastrinis**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "workbench.colorTheme": "Gruber darker (yellow)"
+}
```
Comment (mmcgr) [inline .vscode/settings.json:3]: Accidental include?

**Review by mmcgr — APPROVED** [review]
Looks good, apart from the vscode file.

File: .vscode/settings.json (hunk 1 above)
Comment (gkastrinis) [reply .vscode/settings.json:3]: yep! sorry 😭

Comment (gkastrinis) [reply .vscode/settings.json]: cleaned

**Pull request merged**